### ISSUE TITLE
Store the votes of a user, at least as a hash value.

### ISSUE THREAD

**Issue by AetherPrior** [crux-bphc/course-reviews#1]
Currently, the system involves reading the cookie associated with a site. Anyone can directly clear the cookie and vote twice. :)

**Comment by agrawal-d** [comment]
Can you elaborate what you mean by "atleast as a hash value"?

**Comment by AetherPrior** [comment]
Right, so if storing the relation between a user and their upvotes can cause an issue of privacy, you might hash the combination of the user, the review and their upvote/downvote for that specific review and store the hash instead.  
That way when the user tries to (up/down)vote twice, all you'd want to do is to recompute the hash and perform a lookup.  
If searching large amounts of hash values is a problem, you could store the course ID/user along with the hash itself to potentially minimize amount the lookups.
